data/tech/aria/navigation_role.json

```diff
@@ -9,6 +9,39 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#navigation"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_role",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "div[role=\"navigation\"]",
```

Comment:
This doesn't have to be a `div`. I try to make the CSS target at this level as generic as possible (`[role="navigation"]`).

Reply:
Updated.